src/components/CurrentRoundRow.tsx

```diff
@@ -0,0 +1,174 @@
+  <>
+    <Value>{props.children}</Value>
+    &nbsp;&nbsp;
+    <Unit>KSM</Unit>
+  </>
+)
+
+const Unit = styled.span`
+  color: #6c757d;
+`
+
+const Value = styled.span`
+  color: #fff;
+`
+
+const Title = styled.h4`
+  color: #fff;
+`
+
+const CurrentRoundProgress = (props: { percentageDone: number }): JSX.Element => (
+  <div style={{ width: 100, height: 100 }}>
+    <CircularProgressbar
+      value={props.percentageDone}
+      styles={buildStyles({
+        pathColor: '#E6007A',
+        trailColor: '#fff',
+        strokeLinecap: 'butt',
+      })}
+    />
+  </div>
+)
+
+const CurrentRoundRow = (props: { currentAccount: string }): JSX.Element => {
+  const { api } = useSubstrate()
+  const [currentBlock, setCurrentBlock] = useState<number>(0)
+  const [info, setInfo] = useState<DeriveSociety | any>()
+  const [rotationPeriod, setRotationPeriod] = useState<number>(0)
+  const [strikes, setStrikes] = useState<number | any>(10)
+  const periodBlocksDone = currentBlock % rotationPeriod
+  const periodBlocksLeft = rotationPeriod - periodBlocksDone
+  const percentageDone = 100 - (periodBlocksLeft * 100) / rotationPeriod
+  const [, , time] = useBlockTime(periodBlocksLeft, api)
+  const { days, hours, minutes, seconds } = time
+  const accountBid = '54,223' // TODO: get it from api
+
+  useEffect(() => {
+    if (api) {
+      const rotationPeriod = api.consts.society.rotationPeriod.toNumber()
+      api.derive.chain.bestNumber((block) => {
+        setCurrentBlock(block.toNumber())
+      })
+      setRotationPeriod(rotationPeriod)
+
+      api.derive.society.info().then((response) => {
+        setInfo(response)
+      })
+
+      api.derive.society.members().then((members) => {
+        const account = members.find((member) => member.accountId.toString() === props.currentAccount)
+        setStrikes(account?.strikes.toNumber())
+      })
+    }
+  }, [api])
+
+  return (
+    <Container>
+      <Row>
+        <Col>
+          <Row className="mb-3">
+            <Col>
+              <Title>Current Round</Title>
+            </Col>
+          </Row>
+          <Row>
+            <Col>
+              <CurrentRoundProgress percentageDone={percentageDone} />
+            </Col>
+            <Col style={{ paddingLeft: 0 }}>
+              <Value>{days}</Value>&nbsp;<Unit>day</Unit>
+              <br />
+              <Value>{hours}</Value>&nbsp;<Unit>hrs.</Unit>
+              <br />
+              <Value>{minutes}</Value>&nbsp;<Unit>mins.</Unit>
+              <br />
+              <Value>{seconds}</Value>&nbsp;<Unit>secs.</Unit>
+            </Col>
+          </Row>
+        </Col>
+        <Col>
+          <Row className="mb-3">
+            <Col>
+              <Title>Round Payout</Title>
+            </Col>
+          </Row>
+          <Row>
+            <Col>
+              <FormatedKSM>
+                {formatBalance(info?.pot.toString(), { decimals: 0 }).substring(0, 5).replace('.', ',')}
```

Comment:
this doesn't seem like the right way to do this formatting? the standard in the apps UI is to use system locale iiuc, so Europe will have "," decimals but most places use "." decimals? do you know of any formatter that respects system locale to use instead of doing this? might even be a browser api